### DISCUSSION TITLE
Issue with a pixi class called Image

### DISCUSSION THREAD

**Discussion by NickShreve** [pixijs/pixijs#8601] [Q&A] [answered]
So i am trying to make an image class, that makes a new sprite on the screen to work with some old code that is expecting a class called Image. Issue is, when pixi makes a sprite with something = PIXI.Sprite.from(imageURL/FilePath) later in pixi's code, on line 26 of ImageResource.js, they are making a new Image(). I cant find where that class is defined, any documentation on it, or what it is trying to do. Is there a way PIXI's code can still work but my class can be kept as Image so my old code works?

**Comment by bigtimebuddy** [comment, marked as answer]
`Image` is a browser class, see [documentation](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/Image).

**Reply by NickShreve** [reply]
Ah Thank you... That answers my question. Thanks!
